app/views/vm_common/console_vmrc.html.haml

```diff
@@ -367,36 +367,42 @@
         // Start up VMRC
         vmrc = document.getElementById("vmrc");
 
-        vmNameTooltip = $("#vmNameTooltip");
-        var vmTitle = $("#vmTitle");
+        if (typeof vmrc.isReadyToStart === 'undefined') {
+          window.location.href = "vmrc://clone:#{ticket}@#{host}:443/?moid=#{vmid}";
```

Comment:
You'd need something in javascript that does the right thing.  You would also need to know how the host value is used (sockets or something else) since hostname unwraps the ipv6 value in ruby for use with socket connections (`::1`).  Things like the host value in the http header need to remain wrapped (`[::1]`)